k8s/arroyo/templates/tests/test-grpc-connection.yaml

```diff
@@ -0,0 +1,29 @@
+apiVersion: v1
+kind: Pod
+metadata:
+  name: "{{ include "arroyo.fullname" . }}-compiler-test-grpc-connection"
+  labels:
+    {{- include "arroyo.labels" . | nindent 4 }}
+  annotations:
+    "helm.sh/hook": test
+spec:
+  containers:
+    - name: grpcurl
+      image: fullstorydev/grpcurl:v1.8.7-alpine
+      args: ['-plaintext', '{{ include "arroyo.fullname" . }}-api:{{ .Values.compiler.service.grpcPort }}','list']
```

Comment:
should this be `-compiler` (and below, `-controller`) instead of `-api`?

Reply:
Fixed with
[`af93b05` (#271)](https://github.com/ArroyoSystems/arroyo/pull/271/commits/af93b0544a2a347856846baeae08322681a26d2f)